src/component/mxgraph/shape/gateway-shapes.ts

```diff
@@ -48,31 +62,58 @@ export class ExclusiveGatewayShape extends GatewayShape {
   }
 
   protected paintInnerShape(c: mxgraph.mxXmlCanvas2D, x: number, y: number, w: number, h: number): void {
+    super.paintInnerShape(c, x, y, w, h);
```

Comment:
:question: I see this `super` call in the 2 subclasses, could we remove it with the following
- in the super class, move the code to the enclosing `paintVertexShape` method right before the `paintInnerShape` call
- then we can remove the super calls
- the `paintInnerShape` can then be made abstract

Reply:
managed in f428cbcf779981b337a8682e6824ea81e02cd17e